### ISSUE TITLE
Vertical scroll looks weird in MegaAri(megaari.nes)

### ISSUE THREAD

**Issue by takahirox** [takahirox/nes-rust#11]
[megaari.nes](http://littlelimit.net/fc.htm)

![image](https://user-images.githubusercontent.com/7637832/69312057-42379c80-0c71-11ea-91fe-38d095e68f66.png)

1. No Alice in OP as the above screenshot shows
2. After the OP, it halts with the following message, out of range access of PALETTES array.

```
thread 'main' panicked at 'index out of bounds: the len is 64 but the index is 68', src\ppu.rs:972:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
error: process didn't exit successfully: `target\release\nes-rust.exe ../nes-js/roms2/megaari.nes` (exit code: 101)
```

**Comment by takahirox** [comment]
By #15, the two problems are resolved. But I face a new problem, vertical scrolling looks weird.

**Comment by takahirox** [comment]
Resolved by #19
